xml/_filter.xml

```diff
@@ -148,6 +148,9 @@
     <namespaceFilter name="StackExchange.Redis">
       <typeFitler name="*" expose="false" />
     </namespaceFilter>
+    <namespaceFilter name="System.Text.Json.SourceGeneration">
```

Comment:
Is it possible to filter on assembly name?

Reply:
Not that I know. @TianqiZhang can you confirm? Eric, what is the concern?

Reply:
Well we can just remove the assembly from the reference assemblies that are imported.

Reply:
> Eric, what is the concern?

The concern is someone puts a public API in this namespace in a different assembly. Filtering by namespace would exclude it.

> we can just remove the assembly from the reference assemblies that are imported.

That would be great!

Reply:
There is another assembly that is in the same situation as this one: `Microsoft.Extensions.Logging.Generators`. If it is also imported, could you exclude that assembly as well?

Reply:
> There is another assembly that is in the same situation as this one: `Microsoft.Extensions.Logging.Generators`. If it is also imported, could you exclude that assembly as well?

I'm not seeing Microsoft.Extensions.Logging.Generators.dll in the assembly drop.

Reply:
ok, that's good to know.